modules/microshift-cni-multus-bridge-nw-example-config.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift_multiple_networks/microshift-cni-multus-using.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-cni-multus-bridge-nw-example-config_{context}"]
+= Configuring a bridge-type additional network
+
+After you have created the NetworkAttachmentDefinition object and applied it, use the following example procedure to configure a bridge-type additional network. You can also use this workflow for other additional network types.
+
+.Prerequisite
+. You created and applied the NetworkAttachmentDefinition configuration.
+
+.Procedure
+. If it was not present before, verify that the bridge was created on the host by running the following command:
++
+[source,terminal]
+----
+$ ip a show br-test
+----
++
+.Example output
+[source,terminal]
+----
+22: br-test: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
+    link/ether 96:bf:ca:be:1d:15 brd ff:ff:ff:ff:ff:ff
+    inet6 fe80::34e2:bbff:fed2:31f2/64 scope link
+       valid_lft forever preferred_lft forever
+----
+
+. Configure an IP address for the bridge by running the following command:
++
+[source,terminal]
```

Comment:
The configuration below for creating a bridge is not needed for multus CNI.

Reply:
@pmtk From testing verification steps in https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-72796, QE do not need to add IP for the bridge used by NAD. QE testing is just to check the communication between the pods. It's OK to keep those configuration if you want to check the connectivity between the host and pods.

Reply:
I would prefer to leave the end-to-end docs for now since we have customers doing new configs and figuring things out. We can always reduce later.